### ISSUE TITLE
v4 Input Group height problem

### ISSUE THREAD

**Issue by anthonyvalera** [twbs/bootstrap#17035]
Hey all,

I ran into two minor issues in v4 Input Group documentation and figured I'd put them in one issue.

Input group sizes are styled incorrectly in Chrome Version 44.0.24 and Safari. `input-group-sm` is working in Safari, `input-group-lg` is not.

<img width="882" alt="input groups" src="https://cloud.githubusercontent.com/assets/5510444/9364732/ac0da084-4662-11e5-9090-b7dc15351ab1.png">

Dropdowns on this page have v3 markup.
<img width="881" alt="v4 dropdowns" src="https://cloud.githubusercontent.com/assets/5510444/9365043/3070781e-4664-11e5-883c-9de9287a0a25.png">

I can try to setup and submit a pull-request if needed. v4 looks great btw!

**Comment by rodymol123** [comment]
It appears to be in the button group documentation as well.

**Comment by kkirsche** [comment]
What operating system are you on? As you didn't provide the safari version, I'm not sure what the OS is.

**Comment by rodymol123** [comment]
I'm seeing this in Chrome 44, running Windows 10, but it is most certainly wrong markup ;)
(Looking at the dropdown documentation)

**Comment by anthonyvalera** [comment]
Sorry, OS X Yosemite 10.10.4 and Safari Version 8.0.7 (10600.7.12)

**Comment by rodymol123** [comment]
It seems to be cross browser... also appearing in Edge and Opera.

**Comment by zacechola** [comment]
`.input-group-sm` is actually off a bit, as well.

**Comment by Studio384** [comment]
Dropdowns are broken on multiple places. This happens in every browser I tested (IE, Edge, Firefox and Chrome on Windows and Windows Phone/Windows 10 Mobile).

**Comment by Qqwy** [comment]
I can confirm that this issue also shows up on Chromium(43.0.2357.130) and Firefox(40.0) on Linux( Linux Mint, x64)

The input field has the following padding set: `.input-group-lg>form-control{ padding: .75rem 1.25rem}`, while the .input-group-addon has only the 1.25rem one: `.input-group-lg>.input-group-addon{padding: 1.25rem} `

**Comment by mdo** [comment]
Fixed in https://github.com/twbs/bootstrap/commit/1e7c42d2c086157c77fc8df8dbba3e622c143ed5.

**Comment by tagliala** [comment]
`input-group-sm` is still not working on Chrome Version 44.0.2403.157 (64-bit) OSX 10.10.4

Firefox and Safari are fine

Do we need a separate issue?

**Comment by mdo** [comment]
> Do we need a separate issue?

Nah, just going to re-open this :).

**Comment by KOBA789** [comment]
This issue seems to be caused by Chrome's layout calculation bug.
Chrome renders 31.7813px(=height of `.input-group-sm .form-control`) into not 32px but 31px.
I disabled `float: left` of `.input-group-sm .form-control` then it looks fine for now.

[![Gyazo](https://i.gyazo.com/ad09d91ba9775a8321cfacd3265980a4.gif)](https://gyazo.com/ad09d91ba9775a8321cfacd3265980a4)

**Comment by tagliala** [comment]
Is this something that should be reported on the chromium bug tracker?

**Comment by mdo** [comment]
This is already fixed in the `v4-dev` branch.

**Comment by NKCSS** [comment]
Still have the same issue on Windows 8.1 and IE11

Small demo:

        <!DOCTYPE html>
        <html lang="en">
        <head>
            
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1">
            <meta http-equiv="x-ua-compatible" content="ie=edge">

            <title>Vesta Vastgoed</title>

            
            <link rel="stylesheet" href="https://cdn.rawgit.com/twbs/bootstrap/v4-dev/dist/css/bootstrap.css">
            <link href='https://fonts.googleapis.com/css?family=Roboto:400,700italic,700,400italic,500italic' rel='stylesheet' type='text/css'>
            
            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">

            <body>
                <div class="container">
                    <div class="input-group">
                        <span class="input-group-addon" id="telefoon-uitleg"><i class="fa fa-phone"></i></span>
                        <input type="text" class="form-control" placeholder="Uw nummer" aria-describedby="telefoon-uitleg">
                        <span class="input-group-btn">
                            <button class="btn btn-secondary" type="button">Bel</button>
                        </span>
                    </div>
                </div>
                
                <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
                <script src="https://cdn.rawgit.com/twbs/bootstrap/v4-dev/dist/js/bootstrap.js"></script>
            </body>
        </html>

**Comment by cvrebert** [comment]
JS Fiddle: http://jsfiddle.net/9waquk0b/

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://fiddle.jshell.net/9waquk0b/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [XX0](https://github.com/twbs/bootlint/wiki/XX0): Your example is too large for Bootlint to process! Please simplify your example and make it smaller.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Hmm, that fiddle is definitely not too large; might be an LMVTFY bug...

@NKCSS Confirmed, but the cause seems to be different from this issue, so I opened a new one: #18083. Thanks!
